token/services/identity/sig/sigservice.go

```diff
@@ -181,17 +193,17 @@ func (o *Service) IsMe(identity driver.Identity) bool {
 	}
 
 	// last chance, deserialize
```

Comment:
This is commented out, maybe it's better to remove it?

Reply:
yes